tests/e2e/e2e_test.go

```diff
@@ -23,21 +23,31 @@ import (
 func TestE2E(t *testing.T) {
 	executeTests(t,
 		testExecution{
-			Name:       "TestProfileParsingWorks",
+			Name:       "TestProfileModification",
 			IsParallel: true,
 			TestFn: func(t *testing.T, f *framework.Framework, ctx *framework.Context, mcTctx *mcTestCtx, namespace string) error {
+				const (
+					baselineImage = "quay.io/jhrozek/ocp4-openscap-content@sha256:a1709f5150b17a9560a5732fe48a89f07bffc72c0832aa8c49ee5504510ae687"
+					modifiedImage = "quay.io/jhrozek/ocp4-openscap-content@sha256:7999243c0b005792bd58c6f5e1776ca88cf20adac1519c00ef08b18e77188db7"
```

Comment:
Just wondering if the tag parsing will get the digest properly from a regular tag like `:latest` etc.

Reply:
No, sorry. First, I wasn't sure if this was safe, because the profileparser runs on one node, but the scans run on different. And because the content image is pulled when it's being used by the scans, using the digest is the only way to make sure everything points to the same image (the content provider should in theory be nice and point one versioned tag to one digest and never change it but...). So by using the tag, the profileparser might parse something, then the digest changes because someone pushes a new version of the tag and then the scan uses a different content.. And even on the code side, I wasn't sure how to get the sha hash short of having a cron job that pulls the image periodically and checks the sha itself.

That said, maybe relying on the sha being there is a bit of overkill. Maybe we could instead just use the whole `imageName` regardless of what's there as an annotation? That way we could at least support changing the imageName from e.g. `v0.1` to `v0.2` or such..this still doesn't get away with the race between defining the profilebundle by tag and using the actual content, but maybe it's easier for the admin. I actually start leaning that way. Let me know what you prefer, changing the PR is not a big deal.

Reply:
This is OK for now. if we want to support a regular tag we might be able to mirror images to the internal registry and compare/detect sha changes through the API.